### PR TITLE
Updating template

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

- [X] Updating template
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/genomic-medicine-sweden/tomte/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by Lucpen** [comment]
The container needs to be modified so that it provides a non-root user with a home directory that could be used for the module, however, this is a good patch in the meanwhile.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
 ## Contribution workflow
 
-If you'd like to write some code for tomte, the standard workflow is as follows:
+If you'd like to write some code for genomic-medicine-sweden/tomte, the standard workflow is as follows:
 
-1. Check that there isn't already an issue about your idea in the [tomte issues](https://github.com/genomic-medicine-sweden/tomte/issues) to avoid duplicating work. If there isn't one already, please create one so that others know you're working on this
-2. [Fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) the [tomte repository](https://github.com/genomic-medicine-sweden/tomte) to your GitHub account
+1. Check that there isn't already an issue about your idea in the [genomic-medicine-sweden/tomte issues](https://github.com/genomic-medicine-sweden/tomte/issues) to avoid duplicating work. If there isn't one already, please create one so that others know you're working on this
+2. [Fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) the [genomic-medicine-sweden/tomte repository](https://github.com/genomic-medicine-sweden/tomte) to your GitHub account
+   > > > > > > > TEMPLATE
```
Comment (jemten) [inline .github/CONTRIBUTING.md:20]:
```suggestion
```

File: .github/workflows/release-announcements.yml
Comment (jemten) [inline .github/workflows/release-announcements.yml:1]: I guess this will never really run for us. But lets leave it in

File: .nf-core.yml [hunk 2]
```diff
@@ -20,7 +20,14 @@ lint:
...
+  multiqc_config:
+    - report_comment
   nextflow_config:
     - manifest.name
     - manifest.homePage
-  multiqc_config: false
+repository_type: pipeline
+template:
+  author: lucpen
+  description: Analizing RNAseq data
```
Comment (jemten) [inline .nf-core.yml:31]:
```suggestion
  description: Analyzing RNAseq data
```

File: docs/README.md [hunk 3]
```diff
@@ -1,10 +1,14 @@
...
+
+The genomic-medicine-sweden/tomte documentation is split into the following pages:
 
 - [Usage](usage.md)
   - An overview of how the pipeline works, how to run it and a description of all of the different command-line flags.
 - [Output](output.md)
   - An overview of the different results produced by the pipeline and how to interpret them.
 
-You can find a lot more documentation about installing, configuring and running nf-core and nf-core based pipelines on the website: [https://nf-co.re](https://nf-co.re)
+# You can find a lot more documentation about installing, configuring and running nf-core and nf-core based pipelines on the website: [https://nf-co.re](https://nf-co.re)
```
Comment (jemten) [inline docs/README.md:14]: Should this really be a heading?

File: docs/usage.md [hunk 4]
```diff
@@ -263,6 +265,40 @@ To obtain the gene counts and splice counts you will have to download the counts
...
+```
+
+| Column    | Description                                                                                                                                                                            |
+| --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `sample`  | Custom sample name. This entry will be identical for multiple sequencing libraries/runs from the same sample. Spaces in sample names are automatically converted to underscores (`_`). |
+| `fastq_1` | Full path to FastQ file for Illumina short reads 1. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
+| `fastq_2` | Full path to FastQ file for Illumina short reads 2. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
+
+An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline.
+
```
Comment (jemten) [inline docs/usage.md:301]: I think you can take this part out as it is already covered further up in the document. This is probably only something generic from the template

File: nextflow_schema.json [hunk 5]
```diff
@@ -19,7 +19,7 @@
                     "mimetype": "text/csv",
                     "pattern": "^\\S+\\.csv$",
                     "description": "Path to comma-separated file containing information about the samples in the experiment.",
-                    "help_text": "You will need to create a design file with information about the samples in your experiment before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row. See [usage docs](https://nf-co.re/tomte/usage#samplesheet-input).",
+                    "help_text": "You will need to create a design file with information about the samples in your experiment before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row.",
```
Comment (jemten) [inline nextflow_schema.json:22]: I think we have a requirement of five columns; case, sample, read1, read2 and strandedness

**Review by jemten — COMMENTED** [review]
Nice work figuring out the drop permission errors. A couple of small comments

File: docs/README.md (hunk 3 above)
Comment (Lucpen) [reply docs/README.md:14]: Probably not :)

File: docs/usage.md (hunk 4 above)
Comment (Lucpen) [reply docs/usage.md:301]: You are right :D

File: nextflow_schema.json (hunk 5 above)
Comment (Lucpen) [reply nextflow_schema.json:22]: We do, I forgot to modify there

**Review by jemten — APPROVED** [review]
Looks good!

**Pull request merged**